src/current/v23.2/admission-control.md

```diff
@@ -7,14 +7,36 @@ docs_area: develop
 
 CockroachDB supports an admission control system to maintain cluster performance and availability when some nodes experience high load. When admission control is enabled, CockroachDB sorts request and response operations into work queues by priority, giving preference to higher priority operations. Internal operations critical to node health, like [node liveness heartbeats]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues), are high priority. The admission control system also prioritizes transactions that hold [locks]({% link {{ page.version.version }}/crdb-internal.md %}#cluster_locks), to reduce [contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) and release locks earlier.
 
+[XXX](): check in with #cc-serverless team on the conflicting Serverless statements in here.
+
 {{site.data.alerts.callout_info}}
 Admission control is not available for CockroachDB {{ site.data.products.serverless }} clusters.
 {{site.data.alerts.end}}
 
+## How admission control works
+
+At a high level, the admission control system works by queueing requests to use the following system resources:
+
+- CPU
+- Storage IO (writes to disk)
+
+For CPU, different types of usage are queued differently based on priority to allow important work to make progress even under high CPU utilization.
+
+For storage IO, the goal is to prevent the [storage layer's log-structured merge tree]({% link {{ page.version.version }}/architecture/storage-layer.md %}#log-structured-merge-trees) (LSM) from experiencing high [read amplification]({% link {{ page.version.version }}/architecture/storage-layer.md %}#read-amplification), which slows down reads, while also maintaining the ability to absorb bursts of writes.
+
+Admission control works on a per-[node]({% link {{ page.version.version }}/architecture/overview.md %}#node) basis, since even though a large CockroachDB cluster may be well-provisioned as a whole, individual nodes are stateful and may experience performance [hot spots]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#hot-spots).
+
+For more details about how the admission control system works, see:
+
+- The [Admission Control tech note](https://github.com/cockroachdb/cockroach/blob/master/docs/tech-notes/admission_control.md).
+- The blog post [Here's how CockroachDB keeps your database from collapsing under load](https://www.cockroachlabs.com/blog/admission-control-in-cockroachdb/).
+
```

Comment:
And https://www.cockroachlabs.com/blog/rubbing-control-theory/ which talks about CPU tokens for elastic work.

Reply:
added it! thanks